build.sbt

```diff
@@ -694,8 +687,8 @@ lazy val `scio-core` = project
       "com.github.ben-manes.caffeine" % "caffeine" % caffeineVersion % Provided,
       "com.google.apis" % "google-api-services-dataflow" % googleApiServicesDataflowVersion % Provided,
       "org.apache.beam" % s"beam-runners-flink-$flinkMinorVersion" % beamVersion % Provided,
-      "org.apache.beam" % "beam-runners-google-cloud-dataflow-java" % beamVersion % Provided,
-      "org.apache.beam" % s"beam-runners-spark-$sparkMajorVersion" % beamVersion % Provided,
+      "org.apache.beam" % "beam-runners-google-cloud-dataflow-java" % beamVersion % Provided excludeAll (Exclude.avro),
```

Comment:
why do you excluding avro in this PR?

Reply:
I'm not using `dependencyOverrides` for avro anymore to check the pulled version is the expected one. This means we need to exclude also in non compile scope.

I'm actually wondering if we should go with the flow an let OSS scio upgrade to latest avro WDYT ?

Reply:
Since #5415, no need to evict